src/app/index.ts

```diff
@@ -276,42 +243,54 @@ module.exports = yo.extend({
   writing: {
     copyFiles: function () {
       try {
-        let language = this.project.ts ? 'ts' : 'js';
+        let language = this.project.scriptType  === typescript ? 'ts' : 'js';

```

Comment:
You might want to change this.project.scriptType to this.project.language. For now, we only have typescript/javascript, but it would be more extensible to other languages later. It's not strictly necessary.

Reply:
changing this will require pretty extensive changes throughout index.ts and the test classes. Not sure if it's worth it for now